clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -149,7 +149,11 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
 
     def tags = GCEUtil.buildTagsFromList(description.tags)
 
-    def serviceAccount = GCEUtil.buildServiceAccount(description.authScopes)
+    if (description.authScopes && !description.serviceAccountEmail) {
+      description.serviceAccountEmail = "default"
```

Comment:
It does not. We are just reflecting what the api supports here.
![image](https://cloud.githubusercontent.com/assets/8437718/15397562/a54c4272-1dae-11e6-84e0-9dc85e9c5666.png)